Representable/representable
# fix bug when dragging away from community before saving

**changes**
- fixes bug where if the user drags away from their community before saving, there is an error (https://www.loom.com/share/88b63eca9e1a4c189dcc2651222c1c8e)
- removes deprecated shepherdJS code from geo.js file
**to test**
- python manage.py runserver
- draw community
- drag away so that the community is not at all visible when pressing next
- press next
- check no errors
